.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,69 @@
+<!--
+     For a timely review/response, please avoid force-pushing additional
+     commits if your PR already received reviews or comments.
+
+     Before submitting a Pull Request, please ensure you've done the following:
+     - 📖 Read the MOLE Contributing Guide: https://github.com/csrc-sdsu/mole/blob/master/CONTRIBUTING.md
+     - 📖 Read the MOLE Code of Conduct: https://github.com/csrc-sdsu/mole/blob/master/CODE_OF_CONDUCT.md
+     - 👷‍♀️ Create small PRs. In most cases this will be possible.
+     - ✅ Provide tests for your changes.
+     - 📝 Use descriptive commit messages.
+     - 📗 Update any related documentation and include any relevant screenshots.
+     - 📝 Add license information in the appropriate place. See MATLAB or C++ code for reference.
+      
+          SPDX-License-Identifier: GPL-3.0-or-later
+          © 2008-2024 San Diego State University Research Foundation (SDSURF).
+          See LICENSE file or https://www.gnu.org/licenses/gpl-3.0.html for details. 
+
+     NOTE: Pull Requests from forked repositories may need to be reviewed by
```

Comment:
In general, whenever possible, I try to avoid braking lines in Markdown. And you can replace the "may need" with "need"